force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -176,11 +219,16 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String getCurrentDate() {
         // Get the todays date
-
+        TimeZone zone = UserInfo.getTimeZone();
+        Date dt = Date.today();
+                 
         // Format the current date
+        String formattedDate = dt.month() + '/' + dt.day() + '/' + dt.year();        
```

Comment:
It's good to know you can decompose and work with dates this way, but you can also work with the Datetime.format(format) method. Take a look, it's powerful and will often be easier than this.

Reply:
Thank you. I will look into that.